test/index.js

```diff
@@ -47,19 +47,65 @@ lab.experiment('Postgres Plugin', () => {
 
     lab.test('it registers the plugin', (done) => {
 
+        const realConnect = stub.pg.connect;
+
+        stub.pg.connect = function (connection, callback) {
+
+            stub.pg.connect = realConnect;
+
+            callback(null, {}, () => {});
+        };
+
         server.register(Plugin, (err) => {
 
             Code.expect(err).to.not.exist();
+
             done();
         });
     });
 
 
+    lab.test('it exposes a connect function', (done) => {
```

Comment:
Although this test is contrived since we're stubbing the original `connect` method (not an integration test), I think it's important to demonstrate it's expected use case.